validator_manager/src/exit_validators.rs

```diff
@@ -0,0 +1,625 @@
+
+            // Check validator status after publishing voluntary exit
+            if exit_status {
+                let updated_validator_data = beacon_node
+                    .get_beacon_states_validator_id(
+                        StateId::Head,
+                        &ValidatorId::PublicKey(validator_to_exit),
+                    )
+                    .await
+                    .map_err(|e| format!("Failed to get updated validator details: {:?}", e))?
+                    .ok_or_else(|| {
+                        format!(
+                            "Validator {} is not present in the beacon state",
+                            validator_to_exit
+                        )
+                    })?
+                    .data;
+
+                if validator_data.status == ValidatorStatus::ActiveOngoing
+                    && updated_validator_data.status == ValidatorStatus::ActiveOngoing
+                {
+                    eprintln!("Voluntary exit for Validator {} is waiting to be accepted into the beacon chain.", validator_to_exit);
+                } else if updated_validator_data.status == ValidatorStatus::ActiveExiting {
+                    let exit_epoch = updated_validator_data.validator.exit_epoch;
+                    let withdrawal_epoch = updated_validator_data.validator.withdrawable_epoch;
+
+                    eprintln!("Voluntary exit for validator {} has been accepted into the beacon chain, but not yet finalized. \
+                        Finalization may take several minutes or longer. Before finalization there is a low \
+                        probability that the exit may be reverted.", validator_to_exit);
+                    eprintln!(
+                        "Current epoch: {}, Exit epoch: {}, Withdrawable epoch: {}",
+                        current_epoch, exit_epoch, withdrawal_epoch
+                    );
+                    eprintln!("Please keep your validator running till exit epoch");
+                    eprintln!(
+                        "Exit epoch in approximately {} secs",
+                        (exit_epoch - current_epoch) * spec.seconds_per_slot * E::slots_per_epoch()
+                    );
+                } else if updated_validator_data.status == ValidatorStatus::ExitedSlashed
+                    || updated_validator_data.status == ValidatorStatus::ExitedUnslashed
+                {
+                    {
+                        eprintln!(
+                            "Validator {} has exited at epoch: {}",
+                            validator_to_exit, updated_validator_data.validator.exit_epoch
+                        );
+                    }
+                } else {
+                    eprintln!(
+                        "Validator {} has not initiated voluntary exit. Validator status is: {}",
+                        validator_to_exit, updated_validator_data.status
+                    )
+                }
+            }
+        }
+    }
+
+    if merge {
```

Comment:
Not sure if this `--merge` is useful? I tested pasting the generated JSON on beaconcha.in and it doesn't seem to work. Maybe it is an issue with the generated JSON, or is it because submitting voluntary exit can only be done one at a time?